### DISCUSSION TITLE
What is the difference between the PKsim and Mobi?

### DISCUSSION THREAD

**Discussion by xiangetang** [Open-Systems-Pharmacology/Forum#1090] [Models]
I use those two softwares for the first time, and I don’t know the difference specifically between both of them. I think  Mobi is a advanced software/version of PKsim. We can only use various built-in blocks to set up a PBPK model. If I want to establish a more complicated model, e.g. brain(BBB) or tumor or EHC model, I have to go to Mobi and then revise or add some codes using R or Matlab. Do I understand the relationship between those two softwares correctly? If I don’t, could you please give me a detail explanation?

**Comment by msevestre** [comment]
Hi @xiangetang 
Thanks for checking our suite of tools out. 
A very good place to start would be our comprehensive online documentation

Specifically 
https://docs.open-systems-pharmacology.org/open-systems-pharmacology-suite/modules-philsophy-building-blocks

Everything you need to get started is explained. If anything is not clear or missing, please do not hesitate to ask in the forum.

Enjoy
